test/use-package-test.el

```diff
@@ -1,4 +1,4 @@
-;;; use-package-tests.el --- Tests for use-package.el
+;;; use-package-test.el --- Tests for use-package.el
```

Comment:
Unfortunately ert-runner sees no tests when I use the `-tests.el` suffix, so I'm reverting this rename. The options are to use the `-test.el` suffix or not use ert-runner.

https://travis-ci.org/jwiegley/use-package/builds/239172774#L569